larray/viewer.py

```diff
@@ -2332,34 +2358,50 @@ def new(self):
     def _open_file(self, filepath):
         # XXX : clear console history ?
         self._clear_arrays()
-        session = la.Session(filepath)
+        session = la.Session()
+        if '.csv' in filepath:
+            filepath = [filepath]
+        if isinstance(filepath, (list, tuple)):
+            directory = os.path.dirname(filepath[0])
+            names = [os.path.splitext(os.path.basename(fp))[0] for fp in filepath]
+            session.load(directory, names)
```

Comment:
we probably want to add "something" to open several .csv files in Session. In Session.load maybe?